### PR TITLE
Add decision tree benchmark

### PR DESCRIPTION
**Pull request by MrBago**
This PR moves the mllib-config file to resources and adds a large config file for running larger production benchmarks.

### CODE REVIEW

File: src/main/scala/com/databricks/spark/sql/perf/mllib/MLLib.scala [hunk 1]
```diff
@@ -34,6 +36,21 @@ object MLLib extends Logging {
     e.getCurrentResults()
   }
 
+  private def getConfig(resourcePath: String): String = {
+    val stream = getClass.getResourceAsStream(resourcePath)
+    val buffer = new Array[Byte](10000)
```
Comment (mengxr) [inline src/main/scala/com/databricks/spark/sql/perf/mllib/MLLib.scala]: try `scala.io.Source.fromInputStream(...)`

File: src/main/resources/com/databricks/spark/sql/perf/mllib/config/mllib-large.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+output: /databricks/spark/sql/mllib-perf-ci
+timeoutSeconds: 1000
```
Comment (mengxr) [inline src/main/resources/com/databricks/spark/sql/perf/mllib/config/mllib-large.yaml]: just to confirm, is this timeout good for the large test?

Comment (MrBago) [reply src/main/resources/com/databricks/spark/sql/perf/mllib/config/mllib-large.yaml]: It's enough to benchmark 1 estimator, but we'll need to bump it up as we add more.  I can add a comment.

**Comment by MrBago** [comment]
@mengxr can you take another look.

**Pull request merged**